src/app.service.ts

```diff
@@ -1,7 +1,14 @@
+import { MikroORM } from "@mikro-orm/core";
+import { EntityManager } from "@mikro-orm/postgresql";
 import { Injectable } from "@nestjs/common";
 
 @Injectable()
 export class AppService {
+  constructor(
+    private readonly orm: MikroORM,
+    private readonly em: EntityManager,
+  ) {}
+
```

Comment:
Because we don't end up using these properties, we can probably remove these changes.

Reply:
It might makes sense to put this branch on the back-burner and focus on getting the endpoints written in drizzle first

Reply:
@TangoYankee I'm supportive of that. If you still want to do that, can you convert this PR to draft?